manifests/init.pp

```diff
@@ -148,6 +149,7 @@
     certificate_whitelist             => $certificate_whitelist,
     database_max_pool_size            => $database_max_pool_size,
     read_database_max_pool_size       => $read_database_max_pool_size,
+    automatic_dlo_cleanup             => $automatic_dlo_cleanup,
```

Comment:
We should add a configuration options here for time of day and then use it in the epp template below.